test/data/keymaps/empty-symbol-decl.xkb

```diff
@@ -0,0 +1,5 @@
+xkb_keymap {
+  xkb_symbols "sym" {
+    key <SPC> { [Space] };
+  };
+};
```

Comment:
The test suite currently works because this is not a valid _keymap_ file, only a symbols file. Also, we want to check that the syntax that caused segfault previously is now parsed correctly.

```suggestion
xkb_keymap {
    xkb_keycodes  { include "evdev+aliases(qwerty)" };
    xkb_types     { include "complete" };
    xkb_compat    { include "complete" };
    xkb_symbols "sym" {
        key <SPC> { , [Space] };
    };
};
```

Reply:
Oops, thank you for pointing it out! Fixed.